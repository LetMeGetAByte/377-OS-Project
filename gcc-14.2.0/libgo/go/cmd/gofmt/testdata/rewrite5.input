version https://git-lfs.github.com/spec/v1
oid sha256:9793267228b3c3fc35fad94d9633a7911223175f9395cc55a2b481f02760780b
size 538
