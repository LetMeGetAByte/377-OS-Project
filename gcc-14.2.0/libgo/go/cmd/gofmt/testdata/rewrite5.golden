version https://git-lfs.github.com/spec/v1
oid sha256:77cf60cebe467019f0374fb0ced19ed72e392068ad8af43a05418fdcd66d8560
size 446
